i18n/en.json

```diff
@@ -966,7 +966,7 @@
 	"apihelp-smwtask-param-params": "JSON encoded parameters that match the selected task type requirement",
 	"smw-apihelp-smwtask-example-update": "Example of running a update task for a particular subject:",
 	"smw-api-invalid-parameters": "Invalid parameters, \"$1\"",
-	"smw-parser-recursion-level-exceeded": "The level of $1 recursions was exceeded during a parse process. It is suggested to validated the template structure, or if necessary adjust configuration parameter <code>$maxRecursionDepth</code>.",
+	"smw-parser-recursion-level-exceeded": "The level of $1 recursions was exceeded during a parse process. It is suggested to validate the template structure, or if necessary the adjust configuration parameter <code>$maxRecursionDepth</code>.",
```

Comment:
Shouldn't that be "or if necessary adjust **the**"?

Reply:
:facepalm: 

Fixing.